src/testing/code-debugging.md

```diff
@@ -167,11 +168,17 @@ class AppHome extends StatelessWidget {
 }
 ```
 
-The previous app outputs something like the following
-(the precise details vary by the version of the framework,
-the size of the device, and so forth):
+This application produces the following widget tree.
 
-```
+<details markdown="1">
+<summary><strong>
+Expand the widget tree for Example 1
```

Comment:
Perhaps "Expand to view the" or "Expand for" would be more accurate? Since it's not the widget tree itself being expanded.

```suggestion
Expand to view the widget tree for Example 1
```